devel/lazarus/Portfile

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                lazarus
+version             2.0.4
+revision            0
+categories          devel
+platforms           darwin
+license             GPL-2 LGPL-2
+maintainers         {@kamischi web.de:karl-michael.schindler} openmaintainer
+description         Free Pascal IDE.
+long_description    Lazarus is an open-source development system that builds \
+                    on the Free Pascal compiler by adding an integrated \
+                    development environment (IDE).  It includes a \
+                    syntax-highlighting code editor and a visual form \
+                    designer, as well as a component library that is highly \
+                    compatible with Delphi's Visual Component Library \
+                    (VCL).  The Lazarus Component Library (LCL) includes \
+                    equivalents for many of the familiar VCL controls such \
+                    as forms, buttons, text boxes and so on, which are used \
+                    to create applications with a graphical user interface \
+                    (GUI).
+
+homepage            https://wiki.freepascal.org/Main_Page
+master_sites        sourceforge:lazarus
+set src             ${name}-${version}.tar.gz
+
+checksums           rmd160 72cf729452ef651418d8b5fae8a1094ae9823212 \
+                    sha256 e01d10c1b08fdb9b98fa8f740536add804383e4f39f092059c5e4ac7516c147b \
+                    size   65556293
+
+depends_lib         port:fpc port:fpc-sources
+supported_archs     x86_64
+installs_libs       yes
+universal_variant   no
```

Comment:
I believe that you only need to set `supported_archs     x86_64` here.

Reply:
tried it and you are right. thanks.